editor/SCsub

```diff
@@ -75,30 +75,42 @@ if env.editor_build:
     # Generated with `make include-list` for each resource.
 
     # Editor translations
-    tlist = glob.glob(env.Dir("#editor/translations/editor").abspath + "/*.po")
+    editor_localization_enabled = env["editor_localization"] and os.path.isdir("translations/editor")
```

Comment:
I think it's simpler to do just a `if env["editor_localization"]` covering all three resources.

The path check doesn't seem useful, the folders are in Git so they always exist. If someone deleted them manually, then they can pass that option.